jest-e2e.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   setupFilesAfterEnv: ["./test/e2e/setupEnv.ts"],
   testMatch: ["**/e2e/tests/*.test.ts"],
-  testTimeout: 240_000,
+  testTimeout: 480_000,
```

Comment:
The only question I have here is if there are some tests that will wait this whole time period before failing, because then the whole suite is more likely to time out?

Reply:
Obviously this *can* happen, depending on how slow backend and CI runner are when the tests are run. If it does happen frequently we'll have to use more stripes.

Reply:
I don't think that's necessary: when a test fails it does so due to the timeouts within the test - I've never seen a test "hang" anywhere that made the timeout for the entire test necessary (but I definitely want to keep that timeout just in case such a hang ever occurs).

Reply:
I definitely have seen this, but I think that was like 3 test runners ago 🙃

Reply:
Now I've seen this too, but it wasn't the test taking too long, for some reason the upload of the example recording stalled.